.changeset/violet-dolphins-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@liam-hq/erd-core": patch
+---
+
+🐛 When nodes without a relation are present, set the hidden property to false for the non related group node.
```

Comment:
Please describe the value gained by the user, not the implementation details.
example:

```suggestion
🐛 When nodes without relationships are present, display only that node in RelatedTables.
```

.changeset/late-swans-beam.md seems good to delete.